e2e/testdata/testOCMController/component_version.yaml

```diff
@@ -7,8 +7,8 @@ spec:
   component: component.name.pipeline/podinfo
   interval: 1m
   references:
-    expand: true  
+    expand: true
   repository:
-    url: http://registry.ocm-system.svc.cluster.local:5000
+    url: registry.ocm-system.svc.cluster.local:5000
```

Comment:
This is a necessary change. Otherwise, OCM parses the repo URL as a plain URL and doesn't set the scheme to the appropriate value.

Reply:
but this is coming... we could leave a `todo` here.